### PR TITLE
Added support for Javascript

### PR DESCRIPTION
**Pull request by Vipu97**
Add support for one more  programmming language i.e  javascript.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "private": true,
   "dependencies": {
     "axios": "^0.21.1",
+    "chalk": "^5.3.0",
```
Comment (NegiAkash890) [inline package.json:8]: @Vipu97 please let me know where we are using `chalk` in our project?

File: package.json [hunk 2]
```diff
@@ -24,7 +25,7 @@
     "deploy": "gh-pages -d build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "start": "export SET NODE_OPTIONS=--openssl-legacy-provider && react-scripts start",
+    "start": "set NODE_OPTIONS=--openssl-legacy-provider && react-scripts start",
```
Comment (NegiAkash890) [inline package.json:28]: @Vipu97 pls let me know why this was changed 
`export SET` -> `set`
?

File: src/components/SelectBar.js [hunk 3]
```diff
@@ -75,6 +82,20 @@ const SelectBar = ({ changeLanguage }) => {
...
       />
+      <img
+        src={
+          lang === 'js'
+            ? `${process.env.PUBLIC_URL}/assets/js-active.png`
+            : `${process.env.PUBLIC_URL}/assets/js-inactive.png`
+        }
+        alt="Javascript"
+        width="60px"
+        style={{ backgroundColor: lang === 'js' ? 'white' : 'initial' }}
```
Comment (NegiAkash890) [inline src/components/SelectBar.js:93]: @Vipu97 we aren't using inline styles in our project and for the images we are using two different images one for light mode and the other for dark mode no CSS.

**Review by NegiAkash890 — CHANGES_REQUESTED** [review]
Expect for the comment, rest lgtm
Thanks for contributing

File: package.json (hunk 2 above)
Comment (Vipu97) [reply package.json:28]: the react application was not starting with 'npm start' that's why i changed it for running the project in my local machine and it worked after doing so. I will change it back now

File: src/components/SelectBar.js (hunk 3 above)
Comment (Vipu97) [reply src/components/SelectBar.js:93]: Ok, I will remove it

**Pull request closed without merging**